src/lib.rs

```diff
@@ -1023,4 +1045,30 @@ mod tests {
 			assert_eq!(seed, &mnemonic.to_seed(passphrase)[..], "failed vector: {}", mnemonic_str);
 		}
 	}
+
+        #[test]
+	fn test_finalize_mnemonic() {
+		let vectors = [
+			(
+				"ozone drill grab fiber curtain grace pudding thank cruise elder eight",
+				"about"
+			),
+			(
+				"light rule cinnamon wrap drastic word pride squirrel upgrade then income fatal apart sustain crack supply proud",
+				"access"
+			),
+			(
+				"hamster diagram private dutch cause delay private meat slide toddler razor book happy fancy gospel tennis maple dilemma loan word shrug inflict delay",
+				"balance"
+			),
+                        (
```

Comment:
Is there a reason these parenthesis have different indentation?

Reply:
My IDE wasn't loading the .editorconfig. Fixed it now.